libbeat/outputs/kafka/docs/kafka.asciidoc

```diff
@@ -62,9 +62,9 @@ The cluster metadata contain the actual Kafka brokers events are published to.
 
 ===== `version`
 
-Kafka version {beatname_lc} is assumed to run against. Defaults to 1.0.0.
+Kafka protocol version {beatname_lc} will request when connecting. Defaults to 1.0.0.
 
-Valid values are all kafka releases in between `0.8.2.0` and `2.0.0`.
+Valid values are all kafka releases in between `0.8.2.0` and `2.6.0`.
```

Comment:
```suggestion
Valid values are all kafka releases in between `0.8.2.0` and `2.6.0`.

The protocol version controls the Kafka client features available to {beatname_lc}, it does not preventing {beatname_lc} from connecting to Kafka versions newer than the protocol version.
```
Let's clarify that version above 2.6.0 are likely to work, but we can't use new features added in those versions.

Reply:
@faec how would we know this? sorry I am not sure how to address this comment.

Reply:
The Kafka protocol is known/documented to backwards compatible in general: https://www.confluent.io/blog/upgrading-apache-kafka-clients-just-got-easier/ . Note that is from 2017 so this has been in place for a while.

So a 2.7.0 broker/server should be able to communicate with a client requesting protocol version 2.6.0.

Reply:
@nimarezainia I think we should accept this suggestion and merge the PR.

Reply:
@cmacknz @rdner i believe then the text we have is correct. is it not?

Reply:
Yes, the existing text is not wrong.